templates/dart/lib/services/service.dart.twig

```diff
@@ -1,11 +1,16 @@
 part of {{ language.params.packageName }};
 
 {% macro parameter(parameter) %}
-{% if parameter.required %}required {{ parameter.type | typeName }}{% else %}{{ parameter.type | typeName }}?{% endif %} {{ parameter.name | caseCamel }}{% endmacro %}
+{% if parameter.name == 'queries' %}Query? queries{% else %}
```

Comment:
How safe is depending on the name itself for us? 🤔

Reply:
we did it before for orderType, but we removed that enum